.github/workflows/release.yaml

```diff
@@ -7,22 +7,35 @@ on:
 jobs:
   build_and_release:
     runs-on: ubuntu-latest
+    permissions:
+      packages: write
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
+        uses: docker/setup-qemu-action@v2
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v2
       - name: Login to GitHub Container Registry
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set action link variable
+        run: echo "LINK=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" >> $GITHUB_ENV
       - name: Build and push
         id: docker_build
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v3
         with:
           push: true
-          tags: ghcr.io/kloeckner-i/cloudish-sql:${{ github.event.release.tag_name }}
+          tags: |
+            ghcr.io/kloeckner-i/cloudish-sql:${{ github.event.release.tag_name }}
```

Comment:
there is still kloeckner-i

Reply:
Updated

Reply:
@hyunysmile Could you have a look one more time? :)

Reply:
Maybe @dabde has some time to check it out? 
It's just blocking the Charts PR, and hence the migration 🙃